content/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -45,9 +50,15 @@ The following steps enable plugging in the certificates and key into Citadel:
         --from-file=samples/certs/cert-chain.pem
     {{< /text >}}
 
-1.  Redeploy Citadel, which reads the certificates and key from the secret-mount files by using [Helm](/docs/setup/kubernetes/helm-install/)
+1.  Redeploy Citadel, which reads the certificates and key from the secret-mount files by using Helm
     with `global.mtls.enabled` set to `true` and `security.selfSigned` to `false`.
 
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system -x charts/security/templates/deployment.yaml \
+    --set global.mtls.enabled=true,security.selfSigned=false > $HOME/citadel-plugin-cert.yaml
```

Comment:
I would recommend two separate --set operations here, rather than jamming it into one set operation.  I am not sure one set operation works - it may, but its a little harder to read.

Cheers
-steve

Reply:
Done.

Reply:
is security.selfSigned default to false today?  I don't see any mentioning of it in the install doc being revised above.

Reply:
We do have "By default, Citadel generates self-signed root certificate and key" at the beginning of the doc.